projects/project-player-library/src/lib/shared/start-improvement-popup/start-improvement-popup.component.html

```diff
@@ -0,0 +1,10 @@
+<div class="p-5">
+    <div class="pb-4 text-2xl font-bold">
+        {{ "START_IMPROVEMENT" | translate}} ?
+    </div>
+    <div class="pb-4 font-bold">{{"START_IMPROVEMENT_MSG" | translate }}</div>
+    <div class="pb-4">{{ "START_IMPROVEMENT_MSG_CONFIRM" | translate }}</div>
+    <div class="flex justify-center">
+        <button class="py-3 px-1 primary-button" mat-flat-button (click)="closePopup(true)">{{"JOIN_PROGRAM"| translate}}</button>
```

Comment:
Chnage the button label with the Start improvement

Reply:
done